packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -183,9 +185,41 @@ function isProd(config) {
 		prodConfig.plugins.push(
 			new BabelEsmPlugin({
 				filename: '[name].[chunkhash:5].esm.js',
-				chunkFilename: '[name].chunk.[chunkhash:5].esm.js'
+				chunkFilename: '[name].chunk.[chunkhash:5].esm.js',
+				beforeStartExecution: plugins => {
+					plugins.forEach(plugin => {
+						if (plugin.constructor.name === 'DefinePlugin' && plugin.definitions) {
+							for (const definition in plugin.definitions) {
+								if (definition === 'process.env.ES_BUILD') {
+									plugin.definitions[definition] = true;
```

Comment:
I believe this is the cause of #622

Reply:
AKA, forcing `ES_BUILD` to always be true, even when not flagged. And in `lib/entry.js`, we're looking for `sw-esm.js` when `ES_BUILD` is true.

Reply:
But the Babel esm plugin itself is protected by `config.esm` condition above. So for a normal build it should not execute